### PR TITLE
Modify agent build order

### PR DESCRIPTION
**Pull request by Acuadros95**

### CODE REVIEW

File: .github/workflows/build_agent.yml [hunk 1]
```diff
@@ -36,4 +36,5 @@ jobs:
       run: |
         source /opt/ros/${{ matrix.ros_distribution }}/setup.sh
         cd ros_ws
-        colcon build --cmake-clean-cache --cmake-force-configure --packages-up-to micro_ros_agent microxrcedds_agent --cmake-args -DMICROROSAGENT_SUPERBUILD=OFF
\ No newline at end of file
+        colcon build --cmake-clean-cache --cmake-force-configure --packages-up-to microxrcedds_agent
```
Comment (pablogs9) [inline .github/workflows/build_agent.yml:39]: Don't you need to make a source of `install/local_setup.sh` to add `microxrcedds_agent` to LD_LIRBARY_PATH`?

**Review by pablogs9 — APPROVED** [review]

File: .github/workflows/build_agent.yml (hunk 1 above)
Comment (Acuadros95) [reply .github/workflows/build_agent.yml:39]: Indeed, fixed on 757b4013d34837650955ff7b2f05deedb1de1265

**Comment by Acuadros95** [comment]
@mergigy backport main foxy

**Comment by Acuadros95** [comment]
@mergify backport main foxy

**Pull request merged**